src/probeinterface/probe.py

```diff
@@ -883,10 +893,11 @@ def to_image(self, values, pixel_size=0.5, num_pixel=None, method="linear", xlim
             size of one pixel in micrometers
         num_pixel :
             alternative to pixel_size give pixel number of the image width
-        method : 'linear' or 'nearest' or 'cubic'
-        xlims : tuple or None
+        method : str, default 'linear'
+            One of the options: 'linear' or 'nearest' or 'cubic'
+        xlims : Optional[tuple], default None
```

Comment:
I remember something about default already implying Optionality but I don't remember what Alessio wanted.

Reply:
No that is vscode think. Optional[x] means that it accepts None or X, but optional for vs code means that a default is already supplied. 

This is another reason why I asked my question to Alessio. To try to be consistent in the meaning of optional/Optional.

Reply:
Got you, then fine for me. I don't have a strong opinion on this one.